client/src/components/App.js

```diff
@@ -30,10 +31,11 @@ const App = (props) => {
     <Router>
       <TopBar user={currentUser} />
       <Switch>
-        <Route exact path="/">
-          <h2>Hello from react</h2>
-        </Route>
-        <Route exact path="/restaurants" component={RestaurantList} />
+        <Route exact path="/" component={HomePage} 
+        />
```

Comment:
I think if we put all the props on one line with the `<Route` we should have the closing `/>` on the same line also.
This should be `<Route exact path="/" component={HomePage} />`